diff -urN aa/gnome-themes-2.5.91/configure.in gnome-themes-2.5.91/configure.in
--- aa/gnome-themes-2.5.91/configure.in	2004-03-09 16:03:55.000000000 +0100
+++ gnome-themes-2.5.91/configure.in	2004-03-09 20:19:34.083956496 +0100
@@ -63,7 +63,7 @@
 ### Icon sizes for all large print themes
 
 
-ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA es et eu fa fi fr ga he hi hr hu id is it ja ko li lt lv mk ml mn ms nl no pl pt pt_BR ro ru sk sl sq sr sr@Latn sv th tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA es et eu fa fi fr ga he hi hr hu id is it ja ko li lt lv mk ml mn ms nb nl pl pt pt_BR ro ru sk sl sq sr sr@Latn sv th tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 LARGEPRINT_DND="48,48"
