diff -urN gnome-themes-2.7.91.orig/desktop-themes/Glider/Makefile.am gnome-themes-2.7.91/desktop-themes/Glider/Makefile.am
--- gnome-themes-2.7.91.orig/desktop-themes/Glider/Makefile.am	2004-08-17 19:52:12.226407832 +0200
+++ gnome-themes-2.7.91/desktop-themes/Glider/Makefile.am	2004-08-17 20:03:22.162562056 +0200
@@ -9,7 +9,6 @@
 
 #Keep this hack in until 2.8.0 is released
 install: install-am
-	rm -rf $(datadir)/themes/SimplySmooth
  
 DISTCLEANFILES=index.theme
 EXTRA_DIST=index.theme.in index.theme
diff -urN gnome-themes-2.7.91.orig/gtk-themes/Glider/Makefile.am gnome-themes-2.7.91/gtk-themes/Glider/Makefile.am
--- gnome-themes-2.7.91.orig/gtk-themes/Glider/Makefile.am	2004-08-17 19:52:11.430528824 +0200
+++ gnome-themes-2.7.91/gtk-themes/Glider/Makefile.am	2004-08-17 19:57:40.107562320 +0200
@@ -10,6 +10,5 @@
 
 #Keep this hack in until 2.8.0 is released
 install: install-am
-	        rm -rf $(datadir)/themes/SimplySmooth/gtk-2.0
 
 EXTRA_DIST = $(theme_DATA)
diff -urN gnome-themes-2.7.91.orig/metacity-themes/Makefile.am gnome-themes-2.7.91/metacity-themes/Makefile.am
--- gnome-themes-2.7.91.orig/metacity-themes/Makefile.am	2004-08-17 19:52:11.743481248 +0200
+++ gnome-themes-2.7.91/metacity-themes/Makefile.am	2004-08-17 20:02:59.966936304 +0200
@@ -39,4 +39,3 @@
 
 #Keep this hack in until 2.8.0 is released
 install: install-am
-	        rm -rf $(datadir)/themes/SimplySmooth
