diff -urN gnome-themes-2.26.2/common/all-stock-icons.data gnome-themes-2.26.2.new/common/all-stock-icons.data
--- gnome-themes-2.26.2/common/all-stock-icons.data	2009-05-05 18:21:24.000000000 +0200
+++ gnome-themes-2.26.2.new/common/all-stock-icons.data	2009-05-18 20:40:07.871938114 +0200
@@ -47,7 +47,6 @@
                 media-stop.png media-eject.png \
                 media-play.png media-pause.png media-record.png \
                 media-backward.png media-forward.png \
-		media-backward.png media-forward.png \
 		volume-mute.png volume-zero.png \
 		volume-min.png volume-medium.png volume-max.png \
 		double-click-on.png double-click-off.png double-click-maybe.png \
